docs/getting-started-guides/docker.md

```diff
@@ -162,14 +162,22 @@ See [here](/docs/getting-started-guides/docker-multinode/deployDNS/) for instruc
 
 ### Turning down your cluster
 
-1. Delete all the containers including the kubelet:
+1\. Delete the nginx service and deployment:
```

Comment:
How is this step optional? The other 2 steps are not enough to clean-up enough such that you can run the steps again. This step is required if you want to do that.